### PR TITLE
Update README in preparation for version 1 release

### PR DESCRIPTION
**Pull request by gwaybio**
# Description

Reorganizing, rewording, and adding to README in preparation for version 1 release.

## What is the nature of your change?

- [ ] Bug fix (fixes an issue).
- [x] Enhancement (adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

# Checklist

Please ensure that all boxes are checked before indicating that a pull request is ready for review.

- [x] I have read the [CONTRIBUTING.md](CONTRIBUTING.md) guidelines.
- [x] My code follows the style guidelines of this project.
- [x]  I have performed a self-review of my own code.
- [x]  I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x]  My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have deleted all non-relevant text in this pull request template.

### CODE REVIEW

File: README.md
Comment (kenibrewer) [inline README.md:146]: Install instructions in line 152 should also be changed to match above approach.

**Review by kenibrewer — COMMENTED** [review]
I'd recommend a few changes in the alternative installation instructions:

File: README.md
Comment (gwaybio) [reply README.md:40]: Thanks for this discussion @kenibrewer 

My initial thought with this was to add these instructions for a user scenario where they need new pycytominer functionality. After thinking this through, we should not support this kind of scenario moving forward, after version 1.0, where we should develop a new, more sustainable, release process.

So I will go with your option 1 `Delete the pip install git+git: example entirely` as it is the simplest and supports our long term strategy.

File: README.md
Comment (gwaybio) [reply README.md:135]: yes I believe so. There is another example of this in cell_locations, so I will commit on my end

**Comment by gwaybio** [comment]
Thanks for the review @kenibrewer !

Are you able to approve the PR? It seems you went through thoroughly, so I am happy to merge after having your eyes on it.

**Review by kenibrewer — APPROVED** [review]

**Comment by kenibrewer** [comment]
@gwaybio I approved the changes, but I don't count as a reviewer with write access. I assume you can probably merge anyways though.

**Pull request merged**
